src/conntuple/fivetuple/Factory.h

```diff
@@ -0,0 +1,31 @@
+// See the file "COPYING" in the main distribution directory for copyright.
```

Comment:
I wonder a bit if these could move from `src/conntuple` into  `src/packet_analysis/protocols/ip/conn_key/{vlan,fivetuple}/...`. That's where the `IPBasedConnKey` exists.